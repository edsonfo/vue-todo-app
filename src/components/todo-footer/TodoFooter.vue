<template>
  <v-container>
    <v-card-text class="footer-text">
      Double-click to edit a todo<br />
      Credits: Edson Fernandez<br />
      TodoVueApp
    </v-card-text>
  </v-container>
</template>

<script>
  export default {
    name: 'TodoFooter',

    data: () => ({})
  }
</script>
<style lang="sass" scoped>
  .footer-text
    text-align: center
    color: #bfbfbf
</style>
