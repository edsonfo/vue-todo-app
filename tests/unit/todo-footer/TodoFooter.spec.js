import { shallowMount } from '@vue/test-utils'
import TodoFooter from '@/components/todo-footer/TodoFooter.vue'

describe('TodoFooter component tests', () => {
  const wrapper = shallowMount(TodoFooter, {
    mocks: {
      $t: msg => msg
    }
  })
  it('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
