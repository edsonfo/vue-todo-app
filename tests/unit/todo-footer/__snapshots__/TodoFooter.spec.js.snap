// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoFooter component tests should match snapshot 1`] = `
<v-container-stub
  tag="div"
>
  <v-card-text-stub
    class="footer-text"
  >
    
    Double-click to edit a todo
    <br />
    
    Credits: Edson Fernandez
    <br />
    
    TodoVueApp
  
  </v-card-text-stub>
</v-container-stub>
`;
